.bazelrc

```diff
@@ -23,7 +23,7 @@ test --test_output=errors --test_summary=detailed
 test:ci --color=yes
 test:ci --verbose_failures --test_verbose_timeout_warnings
 test:ci --test_env=GO_TEST_WRAP_TESTV=1
-test:ci --experimental_ui_max_stdouterr_bytes=104857600
+test:ci --experimental_ui_max_stdouterr_bytes=268435456
```

Comment:
Does it a magic number? And what does this control?

Reply:
The maximum size of the stdout / stderr files that will be printed to the console. avoid collecting excessively large log files in CI. But it is too small for developers to check the log.